resource/src/models/index.mustache

```diff
@@ -0,0 +1,9 @@
+const util = require('../utils/module');
+const models = util.slurp('src/models').obj;
+
+// making all the models to be exported pacal cased
+Object.keys(models).forEach(key => {
+  const pascalCasedName = key[0].toUpperCase() + key.substr(1);
+
+  module.exports[pascalCasedName] = models[key];
+});
```

Comment:
two things. it is easier to import things in when you need more than one model somewhere to interact with each other. secondly, this will ensure that they all are initialized at the boot time in rethinkdb